### PR TITLE
NEWS: News update before release

### PR DESCRIPTION
**Pull request by shamisp**
News format was changed slightly to include additional information based on git log merge commits.

### CODE REVIEW

**Comment by shamisp** [comment]
ok to test

File: NEWS [hunk 1]
```diff
@@ -1,18 +1,158 @@
...
+#### UCP
+* Added put/offload/short and put/offload/zcopy protocols
+* Added receive message nbx routine
+* Reworked AM implementation and API, which adds support for RNDV semantics
+* Added support for multi-lane connection manager over TCP
+* Added support for printing AM tls with info log level
+* Implement flush and destroy for UCT EPs on UCP worker
+* Reduced UCP request size
+* Added support for keepalive protocol
+* Added support for multi-fragment
```
Comment (brminich) [inline NEWS:33]: seems we have to be more specific in commit messages to use this new format.
For example this line is not clear (there are others too)

Comment (shamisp) [reply NEWS:33]: More informative and accurate commit message.titles will help. These are very much manually filtered and refined version of the messages. If you have specific suggestion, please go ahead and post it.

File: NEWS [hunk 2]
```diff
@@ -1,18 +1,158 @@
...
+#### UCS
+* Added "inf" and "auto" values to time units
+* Added on-stack constructors for array and string buffer
+* Added ucs_ptr_map_t data structure
+* Added bool CSWAP
+* Improved logging
+* Added optimization for namespace processing
+* Fixes for connection matching functionality
+#### RDMA CORE (IB, ROCE, etc.)
+* Added an option to poll TX CQ every progress iteration
```
Comment (yosefe) [inline NEWS:63]: auto-detect fabric adaptive routing setting

File: NEWS [hunk 3]
```diff
@@ -1,18 +1,158 @@
...
+* Added support for ROCM-3.7+ (warning generated if older version detected)
+#### Architecture
+* Added Arm SVE memcpy()
+* Redesigned Arm WFE support
+* Improved clear_cache performance for Arm
+* Added architecture detection for Zhaoxin CPU
+#### CI
+* Added release builds on CUDA 11
+* Enabled performance validation in gtest
+#### UCP
```
Comment (yosefe) [inline NEWS:24]: prefer using the network interface closes to the currently selected GPU

File: NEWS [hunk 4]
```diff
@@ -1,18 +1,158 @@
...
 ##
 #
 
-## Current
-### Features: TBD
-#### UCX Core
-- Added ucp_tag_msg_recv_nbx routine.
-#### UCX Java (API Preview) TBD
-### Bugfixes: TBD
+## 1.10.0 (February 1, 2021)
```
Comment (yosefe) [inline NEWS:10]: date: TBD

File: NEWS [hunk 5]
```diff
@@ -1,18 +1,158 @@
...
+#### UCP
+* Fixes in datatype support for get_zcopy RNDV
+* Fixes in connection manager disconnect
+* Fixes in assert definitions
+* Fixes in completion flow for failed EP
+* Fixes in flush error handling flow
+* Fixes in latency calculations for wireup protocol
+* Fixes in offload completion with inlined data
+* Fixes in unpacking flow
+#### UCT
```
Comment (yosefe) [inline NEWS:139]: fix performance degradation in UD transport due to excess async events

File: NEWS [hunk 6]
```diff
@@ -1,18 +1,158 @@
...
+* Fixes create STRICT_ORDER key
+#### UGNI
+* Fixing disable logic in config
+* Fixing clang 11 warnings
+#### Java
+* Fixes in build dependencies
+* Fixes in constructing UcpRequest object on error
+* Fixes in exception handling on endpoint closure request
+* Fixes for segfault in UcpErrorHandler
+#### UCP
```
Comment (yosefe) [inline NEWS:130]: fix error handling for different protocols

File: NEWS (hunk 4 above)
Comment (shamisp) [reply NEWS:10]: Actually has to be changed to RC1

**Comment by shamisp** [comment]
@yosefe - I addressed your comments.

**Review by yosefe — APPROVED** [review]

**Pull request merged**